client/components/tinymce/plugins/simple-payments/dialog/product-image-picker.jsx

```diff
@@ -114,6 +114,7 @@ class ProductImagePicker extends Component {
 						enabledFilters={ [ 'images' ] }
 						visible={ isSelecting }
 						isBackdropVisible={ false }
+						disabledDataSources={ [ 'pexels', 'google_photos' ] }
```

Comment:
pexels?

Reply:
Ah the free photo library?

Reply:
I wonder if we should inverse this to enabledSources, with the default being all of them enabled, that would help bulletproof against future changes, should folks add more sources.

Fine to do as a follow up though.

Reply:
That's the actual name:
https://github.com/Automattic/wp-calypso/blob/8b07a94ebd5174df34bc42ae7a76f6cb0e9b29d9/client/my-sites/media-library/filter-bar.jsx#L26
I could maybe switch to constants for readability?

Reply:
> I could maybe switch to constants for readability?

Great idea. :) Fine to do as a follow up.